src/datepicker/datepicker-input.ts

```diff
@@ -185,6 +185,13 @@ export class NgbInputDatepicker implements OnChanges,
    */
   @Output() navigate = new EventEmitter<NgbDatepickerNavigateEvent>();
 
+  /**
+   * An event fired after datepicker closing the datepicker.
```

Comment:
The message here is not that clear

Reply:
My bad, msg changed to: "An event fired after closing datepicker window."